### DISCUSSION TITLE
How to DEBUG Prisma ORM queries

### DISCUSSION THREAD

**Discussion by felinto-dev** [prisma/prisma#9742] [Q&A]
![Screenshot_972](https://user-images.githubusercontent.com/32253743/137112851-1c5117ab-6e60-4546-95c4-31d4ac54952d.png)

JSON filtering is not working as it should. I would like to know which query Prisma sent to the database to find out if the problem was really the way I built that I formatted the WHERE in Prisma or if the problem is in the query generated by Prisma.

However, in the query, the values I passed in my application are hidden. Only $1, $2 appears. How could I configure the prisma logging to show these values?

**Comment by ryands17** [comment]
@felinto-dev 👋 

You cannot directly see the values in the console.I would suggest creating a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this.

**Reply by felinto-dev** [reply]
Hi!

After a lot of trying, I found the solution.
I'll leave the link here to help others.

https://github.com/prisma/prisma/issues/5026#issuecomment-759596097

In the meantime, I created another post in the Prisma discussion area because I'm really lost on how I can solve this problem :-)
If you can help me I will appreciate it a lot!

https://github.com/prisma/prisma/discussions/9743

**Reply by ryands17** [reply]
Answered!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
